recipe/conda_build_config.yaml

```
@@ -1,8 +1,5 @@
 mpi:
-- nompi
-- mpich    # [unix]
-- openmpi  # [unix]
+  - nompi
+  - mpich  # [unix]
+  - openmpi # [unix]
```

Did the renderer forget a space here?
```suggestion
  - openmpi  # [unix]
```